package.json

```diff
@@ -12,7 +12,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
     "babel-plugin-styled-components": "^1.10.7",
-    "bootstrap": "^4.5.0",
+    "bootstrap": "^5.2.0",
```

Comment:
Upgrading from bootstrap v4 to v5 requires updating some components as many of the class names have changed. I was hesitant to increase the version due to this. It shouldn't be too hard to fix however, just need to read through the breaking changes and ctrl-f through this repo.

### See incorrect formating of pictures when compaired to the current live version [here](https://monashhumanpower.org/team)
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/23159604/182504088-d6cb6fd8-f66f-40ba-b324-9ee155720d71.png">

Reply:
Ahhh good catch, but really annoying to find out this :(